transform_version causing problems

```
while executing (to_field "transform_version" at traject_configs/sakip_sabanci_common_config.rb:32)

    Record: <?xml version="1.0"?>
<record xmlns="http://www.openarchives.org/OAI/2.0/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
  <header>
    <identifier>oai:cdm21044.contentdm.oclc.org:abidindino/704</identifier>
    <datestamp>2014-01-09</datestamp>
    <setSpec>abidindino</setSpec>
  </header>
  <metadata>
    <oai_dc:dc xmlns:oai_dc="http://www.openarchives.org/OAI/2.0/oai_dc/" xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.openarchives.org/OAI/2.0/oai_dc/ http://www.openarchives.org/OAI/2.0/oai_dc.xsd">
      <dc:type>Yaz&#x131;&#x15F;ma</dc:type>
      <dc:title>Kandilli, 13 Haziran 1979 tarihli Abidin Dino'ya mektup.</dc:title>
      <dc:language>T&#xFC;rk&#xE7;e</dc:language>
      <dc:creator>Edg&#xFC;, Ferit, 1936-</dc:creator>
      <dc:description>Bodrum - Fethi Naci - Ferit Edg&#xFC; - Vakko - Vitali Hakko - Leyla Umar - "Cumhuriyet" - Rasih Nuri &#x130;leri. Mektup.</dc:description>
      <dc:format>pdf</dc:format>
      <dc:identifier>http://digital.sabanciuniv.edu/abidindino/mektup/3040200000988.pdf</dc:identifier>
      <dc:rights>&#xA9; Sabanc&#x131; &#xDC;niversitesi Sak&#x131;p Sabanc&#x131; M&#xFC;zesi. T&#xFC;m haklar&#x131; sakl&#x131;d&#x131;r.</dc:rights>
      <dc:identifier>http://cdm21044.contentdm.oclc.org/cdm/ref/collection/abidindino/id/704</dc:identifier>
    </oai_dc:dc>
  </metadata>
</record>

    Exception: Errno::ENOENT: No such file or directory - git
    /opt/traject/lib/macros/version.rb:11:in ``'

[ERROR] No such file or directory - git
/opt/traject/lib/macros/version.rb:11:in ``': No such file or directory - git (Errno::ENOENT)
	from /opt/traject/lib/macros/version.rb:11:in `block in version'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer/step.rb:138:in `block in execute'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer/step.rb:135:in `each'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer/step.rb:135:in `execute'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:464:in `block (2 levels) in map_to_context!'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:504:in `handle_mapping_errors'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:463:in `block in map_to_context!'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:457:in `each'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:457:in `map_to_context!'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/indexer.rb:582:in `block (3 levels) in process'
	from /usr/local/bundle/gems/traject-3.2.0/lib/traject/thread_pool.rb:123:in `block in maybe_in_thread_pool'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:348:in `run_task'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:337:in `block (3 levels) in create_worker'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `loop'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:320:in `block (2 levels) in create_worker'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `catch'
	from /usr/local/bundle/gems/concurrent-ruby-1.1.5/lib/concurrent/executor/ruby_thread_pool_executor.rb:319:in `block in create_worker'
```

I have to comment out the `to_field transform_versian` line to get any config to run

--- a/lib/macros/version.rb
+++ b/lib/macros/version.rb
@@ -8,7 +8,7 @@ module Macros
     # @return [Proc] a proc that traject can call for each record
     def version
       lambda do |_, accumulator|
-        accumulator << ENV.fetch('VERSION', `git rev-parse --short HEAD`.strip)
+        accumulator << ENV.fetch('VERSION') { `git rev-parse --short HEAD`.strip }
       end
     end
   end

